Dockerfile

```diff
@@ -47,6 +47,8 @@ COPY --from=build /tools /opt/dotnetcore-tools
 EXPOSE 104
 EXPOSE 5000
 
+RUN mkdir /database/
```

Comment:
This is mapped into the container, and that's why we don't create it in the Dockefile.

Reply:
Right ok, so this should be a volume mount point then.

Reply:
Right, it's a mount point. Can we remove this so that it doesn't become a silent failure?